wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -1,117 +1,52 @@
+- **examples of drug use, preparation, or identification with any illicit drug subculture**
+- **depictions of any deliberately inflammatory political, cultural, religious or social content**
 
-## Further considerations
+## Exceptions
 
-The anecdotal "rule of thumb" is that one should generally be comfortable with their family members or other judgemental authority figure viewing the content and be capable of adequately explaining what is going on in it.
+Images or visual elements containing examples of the following may be considered acceptable:
 
-This is by no means a strict requirement, but it is a good way to help inform your decisions on what is and isn't appropriate.
+- mild to moderate depictions of violence (small amounts of blood, non-critical or superficial wounding)
+- artistic nudity (in the context of classical artwork, or work related to the element, such as album art with minor nudity present)
+- mild romance, flirting or intimate affection (hugging, embracing, kissing without sexual connotations, hand-holding, etc)
+- swimsuits of reasonable design (standard swimwear with general coverage, ordinary bikini)
 
-### What do I do if I am uncertain about a piece of visual content?
+## Getting your image assessed
 
-In situations whether the appropriateness of a given piece of visual content is unclear, the mapper must submit a formal request to any Beatmap Nominator for further appraisal, who will then refer the issue to the Global Moderation Team to decide based on a supermajority (>70%) consensus.
+If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) or member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).**
 
-Once a consensus is met and issued, the resulting ruling on its appropriateness is final and can not be challenged under any circumstances, unless edits are made to try and fix the contentious issues cited by the ruling. Any number of these subsequent requests may be made within reason, so long as a good faith effort is being made to try and fix the issues listed.
+The image will be raised to a vote that must pass by a considerable majority, or will be automatically denied. The result of any vote is **final** and may not be overridden or raised for discussion again, unless the image is significantly altered in an attempt to resolve the problems with it.
 
-Should the alteration exceed the limits of common reason or is deemed to be performed in bad faith (such as excessive or poorly-done censoring), the content may be denied from further appraisal requests and thus deemed unusable for use in a beatmap. This is non-contestable and irreversible.
+Any images used elsewhere on osu! (avatars, profile covers, userpage images, etc) are not subject to the voting process outlined above, and are instead left to the discretion of the Global Moderation Team.
```

Comment:
kinda feels like this should come first in the section, since it's more general and seems expected that GMT handle this stuff. then you can go on to mention how beatmaps are a special case with the voting and BN involvement yada yada